python/tvm/meta_schedule/cost_model/xgb_model.py

```diff
@@ -573,22 +592,19 @@ def rmse(ys_pred: np.ndarray, d_train: "xgb.DMatrix"):  # type: ignore # pylint:
         def avg_peak_score(ys_pred: np.ndarray, d_train: "xgb.DMatrix"):  # type: ignore # pylint: disable = unused-argument
             return self.d_train.average_peak_score(ys_pred, self.average_peak_n)
 
+        xgb_custom_callback = XGBoostCustomCallback(
+            early_stopping_rounds=self.early_stopping_rounds,
+            verbose_eval=self.verbose_eval,
+            fevals=[rmse, avg_peak_score],
+            evals=[(self.d_train.dmatrix, "tr")],
+            cvfolds=None,
+        )
```

Comment:
put it back to a few lines below

Reply:
I think we may not be able to move this down because it’s used as parameter in `callbacks=[xgb_custom_callback]`, wdyt?